doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+
+While both DECT NR+ and LTE address different needs, DECT NR+ offers distinct advantages in specific areas, as summarized by the following table:
+
+.. table:: Feature comparison of DECT NR+ and LTE
+
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Feature                               | DECT NR+                          | LTE                        |
+   +=======================================+===================================+============================+
+   | License                               | License-exempt 1.9 GHz            | Requires licenses (varies) |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | IP based                              | No                                | Yes                        |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Coverage                              | Shorter, up to 3 km               | Wider, greater than 10 km  |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Performance in congested environments | Better                            | Poor                       |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Network flexibility                   | Star or mesh networks             | Primarily carrier-provided |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Power Consumption                     | Lower                             | Higher                     |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Data Rate (max)                       | 3.4 Mbps                          | Varies (typically higher)  |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Latency                               | Lower                             | Higher                     |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Security                              | Strong encryption (AES-128)       | Encryption available       |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Network Hiding                        | Possible                          | Not available              |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Focus                                 | mMTC, URLLC                       | General purpose            |
+   +---------------------------------------+-----------------------------------+----------------------------+
+
```

Comment:
Should add the statement about SIM card, not needed at all in DECT NR+

Reply:
@divipillai can you please add a row about that?